lib/issue-to-line/utils.ts

```diff
@@ -43,8 +43,8 @@ export function findLineNumberOfGivenPath(
     (node) => node.key === pathDetails.path[0],
   );
   if (filteredNodes.length === 0) {
-    //Not exists
-    return nodes[0].lineLocation.line;
+    // If the path does not exist, we will return '-1'
+    return -1;
```

Comment:
Nice, what do you think about doing this for line 73 also?

Reply:
🤦🏼‍♀️